apps/etterlatte-brev-api/src/main/kotlin/no/nav/etterlatte/brev/model/tilbakekreving/TilbakekrevingBrevData.kt

```diff
@@ -31,6 +34,9 @@ data class TilbakekrevingBrevDTO(
                 innhold = redigerbart,
                 sakType = generellBrevData.sak.sakType,
                 bosattUtland = generellBrevData.utlandstilknytning?.type == UtlandstilknytningType.BOSATT_UTLAND,
+                brukerNavn = generellBrevData.personerISak.soeker.formaterNavn(),
+                // TODO hvordan vet vi om det er dødsbo?
```

Comment:
Reknar med du eller nokon har planar for desse TODO-ane.

Reply:
@madsop-nav Det kommer inn endringer fra en branch jeg har som inneholder felter for begge disse verdiene.